EasyPost/EasyPost.csproj

```diff
@@ -66,7 +66,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="[13.0.1, 14.0.0)" />
-    <PackageReference Include="RestSharp" Version="[108.0.1, 109.0.0)" />
+    <PackageReference Include="RestSharp" Version="110.2.0" />
```

Comment:
This is a major breaking change since it'll require 110 and we are backporting this. I'd suggest instead that maybe we simply remove the max pin and keep the min pin. That way it should continue to work for folks until RestSharp break something bad.

Reply:
Oops, that was my original intention. Seems my IDE removed the range when I actually went to update RestSharp. This has been rectified.